project/src/test/java/com/google/sps/servlets/AuthServletTest.java

```diff
@@ -1,80 +1,80 @@
-// Copyright 2021 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// // Copyright 2021 Google LLC
```

Comment:
Did you mean to commit this file?

Reply:
mb, that was a side effect from the branch this was branched off

Reply:
Fixed.